probeinterface/probe.py

```diff
@@ -620,7 +623,8 @@ def from_dataframe(df):
             shape_params=shape_params)
 
         if 'device_channel_indices' in df.columns:
-            probe.set_device_channel_indices(df['device_channel_indices'].values)
+            dev_channel_indices = df['device_channel_indices'].replace('', '-1')
```

Comment:
dev_channel_indices is supposed to be integer index.

Reply:
Ok, I didn't know. I added the corresponding restriction in `probe.set_device_channel_indices`, so it should be consistent independent of the origin of the indices now. Here in this line all input is still in string format, so I wouldn't mix string and int here.